### ISSUE TITLE
NPE on ftp setting

### ISSUE THREAD

**Issue by Bambooin** [TeamAmaze/AmazeFileManager#2434]
**Describe the bug**
NPE on ftp setting in last recent build

On 3.5.3 it works, but break on 3.6.0

**To Reproduce**
1. Go to 'FTP Server'
2. Click on right menu
3. Select one of submenu like: 'FTP port'

**Expected behavior**
FTP setting

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: [Redmi Note 7 Pro]
 - OS: [LineageOS 16.0]
 - Rooted: [No]
 - Version: [v3.6.0]

**Additional context**

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
	at com.amaze.filemanager.ui.activities.MainActivity.onOptionsItemSelected(MainActivity.java:1000)
	at android.app.Activity.onMenuItemSelected(Activity.java:3544)
	at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:384)
	at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:228)
	at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
	at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
	at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:65)
	at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:207)
	at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:779)
	at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:834)
	at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
	at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:985)
	at androidx.appcompat.view.menu.MenuPopup.onItemClick(MenuPopup.java:128)
	at android.widget.AdapterView.performItemClick(AdapterView.java:318)
	at android.widget.AbsListView.performItemClick(AbsListView.java:1159)
	at android.widget.AbsListView$PerformClick.run(AbsListView.java:3136)
	at android.widget.AbsListView$3.run(AbsListView.java:4052)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

```

**Comment by TranceLove** [comment]
Side effect after #2293. Fix is at #2341 which is still under review.

**Comment by Bambooin** [comment]
Thanks for your work, as I rely ftp to transfer file between my computer and phone heavily.
